components/src/status_im/ui/components/react.cljs

```diff
@@ -68,6 +68,12 @@
 (def activity-indicator (get-class "ActivityIndicator"))
 
 (def modal (get-class "Modal"))
+(def popup-menu-provider (adapt-class (.-MenuProvider js-dependencies/popup-menu)))
+(def popup-menu (adapt-class (.-Menu js-dependencies/popup-menu)))
+(def popup-menu-options (adapt-class (.-MenuOptions js-dependencies/popup-menu)))
+(def popup-menu-option (adapt-class (.-MenuOption js-dependencies/popup-menu)))
+(def popup-menu-trigger (adapt-class (.-MenuTrigger js-dependencies/popup-menu)))
+(def popup-menu-renderers (js->clj (.-renderers js-dependencies/popup-menu) :keywordize-keys true))
```

Comment:
I'm lacking context here clearly, but what is the guideline for converting to clj data structure in the last form versus `adapt-class` in the others? Thank you!

Reply:
`renderers` is not a class, but a folder inside `react-native-popup-menu`. In this case `(.-renderers js-dependencies/popup-menu)` returns a js object with string keys (map of class names to objects) So we convert it to a Clojure-style keyword map.

Reply:
great, thanks for clarifying.